rhel7/profiles/hippa.xml

```diff
@@ -0,0 +1,17 @@
+<Profile id="hippa">
+<title override="true">Health Insurance Portability and Accountability Act (HIPPA) Security Rule Profile</title>
+<description override="true">
+The HIPAA Security Rule establishes national standards to protect individuals’
```

Comment:
Would this be "HIPPA Security Rule" or "HIPPA profile" ?

Reply:
It is actually referred to as the "HIPPA Security Rule"